<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayer</class>
 <widget class="QWidget" name="VideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255,0);</string>
  </property>
  <widget class="QWidget" name="tilteWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>70</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));</string>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>视频</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="return_btn">
    <property name="geometry">
     <rect>
      <x>946</x>
      <y>0</y>
      <width>78</width>
      <height>70</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>78</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(255, 255, 255， 0);
border-image: url(:/images/playerIcons/return_d.gif);
}
QPushButton:pressed{
	border-image: url(:/images/playerIcons/return_u.gif);
}
QPushButton:hover{


}
QPushButton:released{
	border-image: url(:/images/playerIcons/return_d.gif);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="controlButtonwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>525</y>
     <width>1024</width>
     <height>75</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>75</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="subtitleSelectButton">
      <property name="minimumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}

QPushButton:pressed{
border-radius:5px;
	border:3px solid red;
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));
}

QPushButton:released{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../image.qrc">
        <normaloff>:/images/playerIcons/settings_new(1).png</normaloff>:/images/playerIcons/settings_new(1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fullScreenButton">
      <property name="minimumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}

QPushButton:pressed{
border-radius:5px;
	border:3px solid red;
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));
}

QPushButton:released{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/playerIcons/fullscreenNormal.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="preVideoButton">
      <property name="minimumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}

QPushButton:pressed{
border-radius:5px;
	border:3px solid red;
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));
}

QPushButton:released{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/playerIcons/preVideo.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="playButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}

QPushButton:pressed{
border-radius:5px;
	border:3px solid red;
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));
}

QPushButton:released{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/playerIcons/Play.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextVideoButton">
      <property name="minimumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}

QPushButton:pressed{
border-radius:5px;
	border:3px solid red;
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));
}

QPushButton:released{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/playerIcons/nextVideo.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="minimumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}

QPushButton:pressed{
border-radius:5px;
	border:3px solid red;
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));
}

QPushButton:released{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/playerIcons/Search.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="playListButton">
      <property name="minimumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>146</width>
        <height>75</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}

QPushButton:pressed{
border-radius:5px;
	border:3px solid red;
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));
}

QPushButton:released{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(40, 15, 166), stop:1 rgb(14, 47, 136));
border-radius:5px;
border:0px soild red;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/playerIcons/list.png</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="subtitlewidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>1024</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>60</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="2" column="0">
     <widget class="QLabel" name="subtitleLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="curTimeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);
color: white;
</string>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="progressSlider">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>820</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider{
border-color: #bcbcbc;
background-color:rgba(255,255,255,0);
}
QSlider::groove:horizontal {                                
     border: 1px solid #999999;                             
     height: 1px;                                           
    margin: 0px 0;                                         
     left: 5px; right: 5px; 
 }
QSlider::handle:horizontal {                               
     border: 0px ;                           
     border-image: url(:/images/cameraIcons/silderController.png);
     width: 15px;                                           
     margin: -7px -7px -7px -7px;                  
} 
QSlider::add-page:horizontal{
background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #bcbcbc, stop:0.25 #bcbcbc, stop:0.5 #bcbcbc, stop:1 #bcbcbc); 

}
QSlider::sub-page:horizontal{                               
 background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #439cf3, stop:0.25 #439cf3, stop:0.5 #439cf3, stop:1 #439cf3);                     
}</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalTimeLabel">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);
color:white;</string>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="UnFullscreenButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgba(0, 255, 255,120);
border-radius:15px;
border:0px soild red;
}
QPushButton:pressed{
border-radius:15px;
	border:0px solid red;
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));
}
QPushButton:hover{
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));
border-radius:15px;
	border:0px solid red;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/images/playerIcons/fullscreenPressed.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="sliderAndTimeLabelWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>490</y>
     <width>1024</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>30</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="videoNameWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>512</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>512</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>512</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255,0);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,5">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QLabel" name="videoNameLabel">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255,0);
color:rgb(170, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#aaffff;&quot;&gt;当前正在播放：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>controlButtonwidget</zorder>
  <zorder>subtitlewidget</zorder>
  <zorder>UnFullscreenButton</zorder>
  <zorder>sliderAndTimeLabelWidget</zorder>
  <zorder>videoNameWidget</zorder>
  <zorder>tilteWidget</zorder>
 </widget>
 <resources>
  <include location="../../image.qrc"/>
 </resources>
 <connections/>
</ui>
